Sources/DeploymentTargetAWSLambdaRuntime/LambdaRuntime.swift

```diff
@@ -52,4 +57,101 @@ public class LambdaRuntime: DeploymentProviderRuntime {
         }
         return .invokeDefault(url: url)
     }
+    
+    public static var exportCommand: StructureExporter.Type {
+        LambdaStructureExporterCommand<Service>.self
+    }
+    
+    public static var startupCommand: DeploymentStartupCommand.Type {
+        LambdaStartupCommand<Service>.self
+    }
+}
+
+
+public struct LambdaStructureExporterCommand<Service: WebService>: StructureExporter {
```

Comment:
The subcommands and their properties can be made private or moved to an other file (maybe cleaner) and can be made internal.

Reply:
The intension to make the subcommands public was that if the user/ developer manually overrides the `CommandConfiguration` in its main file, but still wants to support deployment, he needs to add the commands himself. But I agree, the commands could be moved to separate files.

Reply:
I see, makes sense ... maybe we should add some documentation to all public declarations so a user is informed about the general purpose of the command and if/how the developer would use it if it is manually needed 👍